setup.cfg

```diff
@@ -39,7 +39,8 @@ mypy_path = ./stubs
 ignore_missing_imports = True
 show_error_codes = True
 warn_redundant_casts = True
-warn_unused_ignores = True
+# skipping unused ignores bc of numpy v difference between py37 and >=py38
```

Comment:
I guess it depends on the version we run the mypy checks, no? If we run on 3.8 or 3.9, we should have to do that, correct?

Reply:
We are running the mypy checks on 3.9 and if we upgrade to 1.23.x numpy right away then we could potentially get rid of this and also the warnings mentioned above. Let me see... 1.23.x is also required for the tf 2.10 upgrade on M1 macs. So I'll give it a shot

Reply:
The downside is that running mypy on py37 is bound to fail then

Reply:
> The downside is that running mypy on py37 is bound to fail then

yes, but at this stage our CI doesn't enforce mypy checks to go green on every Python version. So this wouldn't only be failing locally if a dev runs the checks on py3.7. I'd say it's a small caveat

Reply:
I reinstated those warnings, as I removed all the type ignore comments anyway with the upgrade to 1.23.x